### PR TITLE
AccountUpdate.Steps definition is created and Modified account scenarios

### PR DESCRIPTION
**Pull request by diogoti2005**

### CODE REVIEW

File: src/test/java/com/demoqa/shop/steps/UpdateAccountStepdefs.java [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+    Logger log = LoggerFactory.getLogger(UpdateAccountStepdefs.class);
+    public WebDriver driver = Browser.getBrowser();
+
+    @Given("Demoqa shop homepage is opened")
+    public void demoqaShopHomepageIsOpened() {
+        HomePage homePage = (HomePage) ScenarioContext.getInstance().getContext(Context.CURRENT_PAGE);
+        ATFAssert.assertEquals(homePage.getTitle(), driver.getTitle(), "New page is opened", "Page" + homePage.getUrl() + " is opened");
+
+        // assertEquals(homePage.getTitle(), driver.getTitle());
```
Comment (en-rmrosca) [inline src/test/java/com/demoqa/shop/steps/UpdateAccountStepdefs.java]: remove commented code if not necessary

File: src/test/java/com/demoqa/shop/steps/UpdateAccountStepdefs.java [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+                homePage.getLnk_MyAccount().getText());
+        ScenarioContext.getInstance().setContext(Context.CURRENT_PAGE, new AccountPage(driver));
+    }
+
+    @And("my-account page is opened")
+    public void myAccountPageIsOpened() {
+        AccountPage accountPage = (AccountPage) ScenarioContext.getInstance().getContext(Context.CURRENT_PAGE);
+        ATFAssert.assertEquals(accountPage.getTitle(), driver.getTitle(), "New page is opened", "Page" + accountPage.getUrl() + " is opened");
+
+        //   assertEquals(accountPage.getTitle(), driver.getTitle());
```
Comment (en-rmrosca) [inline src/test/java/com/demoqa/shop/steps/UpdateAccountStepdefs.java]: same as above

File: src/test/java/com/demoqa/shop/steps/UpdateAccountStepdefs.java [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+
+    @And("submits his request in registry form")
+    public void submitsHisRequestInRegistryForm() {
+        AccountPage accountPage = (AccountPage) ScenarioContext.getInstance().getContext(Context.CURRENT_PAGE);
+        accountPage.getButton_register().isSelected();
+        accountPage.getButton_register().sendKeys(Keys.ENTER);
+        log.info("Button register is clicked.");
+        ScenarioContext.getInstance().setContext(Context.CURRENT_PAGE, new LoginPage(driver));
+
+
```
Comment (en-rmrosca) [inline src/test/java/com/demoqa/shop/steps/UpdateAccountStepdefs.java:82]: remove empty spaces

File: src/test/java/com/demoqa/shop/steps/UpdateAccountStepdefs.java [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+    public void demoqaShopHomepageIsOpened() {
+        HomePage homePage = (HomePage) ScenarioContext.getInstance().getContext(Context.CURRENT_PAGE);
+        ATFAssert.assertEquals(homePage.getTitle(), driver.getTitle(), "New page is opened", "Page" + homePage.getUrl() + " is opened");
+
+        // assertEquals(homePage.getTitle(), driver.getTitle());
+        log.info("Home page with url {}  is loaded",
+                driver.getCurrentUrl());
+    }
+
+    @And("he clicks account link menu")
```
Comment (en-rmrosca) [inline src/test/java/com/demoqa/shop/steps/UpdateAccountStepdefs.java]: use neutral "user" instead of "he" or "she"

Comment (en-rmrosca) [reply src/test/java/com/demoqa/shop/steps/UpdateAccountStepdefs.java]: make the same changes for the rest of the steps

File: src/test/java/com/demoqa/shop/steps/UpdateAccountStepdefs.java [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+                newName);
+    }
+
+    @And("submits save request")
+    public void submitsSaveRequest() {
+        AccountEditPage accountEditPage = (AccountEditPage) ScenarioContext.getInstance().getContext(Context.CURRENT_PAGE);
+        accountEditPage.getButton_save_account().click();
+        log.info(" Save request is submitted on registry form. ");
+        ScenarioContext.getInstance().setContext(Context.CURRENT_PAGE, new AccountPage(driver));
+
```
Comment (en-rmrosca) [inline src/test/java/com/demoqa/shop/steps/UpdateAccountStepdefs.java]: remove empty space

File: src/test/resources/features/AccountUpdate.feature [hunk 6]
```diff
@@ -11,36 +11,32 @@ Feature: Modify account
 
     Given customer enters his new username, email and password in registry form
     And submits his request in registry form
-    Then Demoqa shop  login page is opened
+    Given Demoqa shop  login page is opened
     When he clicks back to home
-    And clicks account link menu
+    And he clicks account link menu
```
Comment (en-rmrosca) [inline src/test/resources/features/AccountUpdate.feature]: Also here change all "he" to "user"

File: src/test/java/com/demoqa/shop/steps/UpdateAccountStepdefs.java (hunk 1 above)
Comment (diogoti2005) [reply src/test/java/com/demoqa/shop/steps/UpdateAccountStepdefs.java]: commented code is removed

File: src/test/java/com/demoqa/shop/steps/UpdateAccountStepdefs.java (hunk 2 above)
Comment (diogoti2005) [reply src/test/java/com/demoqa/shop/steps/UpdateAccountStepdefs.java]: commented code is removed

File: src/test/java/com/demoqa/shop/steps/UpdateAccountStepdefs.java (hunk 4 above)
Comment (diogoti2005) [reply src/test/java/com/demoqa/shop/steps/UpdateAccountStepdefs.java]: "he" was relapsed with "user"

File: src/test/java/com/demoqa/shop/steps/UpdateAccountStepdefs.java (hunk 5 above)
Comment (diogoti2005) [reply src/test/java/com/demoqa/shop/steps/UpdateAccountStepdefs.java]: empty spaces were removed

File: src/test/resources/features/AccountUpdate.feature (hunk 6 above)
Comment (diogoti2005) [reply src/test/resources/features/AccountUpdate.feature]: "he" was replaced with "user"

**Review by en-nlasco — APPROVED** [review]

**Review by en-rmrosca — APPROVED** [review]

**Review by catalindabija — APPROVED** [review]

**Pull request merged**